Luke_Smith_Portfolio_About_Me.html

```diff
@@ -0,0 +1,15 @@
+<body>
```

Comment:
There are a few errors thrown by the validator which I suspect are persistent across most of your pages:
- you need <!DOCTYPE html> at the start of the page
- you don't seem to have an <html> element. In particular, the validator is complaining that you haven't declared a "lang" attribute.
- this webpage doesn't have a title. You'll need to add a <title> element to the page's head.